doc/rados/configuration/ceph-conf.rst

```diff
@@ -550,20 +544,29 @@ or:
       "can_update_at_runtime": false
   }
 
-The ``level`` property can be any of `basic`, `advanced`, or `dev`.
-The `dev` options are intended for use by developers, generally for
-testing purposes, and are not recommended for use by operators.
+The ``level`` property can be ``basic``, ``advanced``, or ``dev``.  The `dev`
+options are intended for use by developers, generally for testing purposes, and
+are not recommended for use by operators.
 
+.. note:: This command uses the configuration schema that is compiled into the
+   running monitors. If you have a mixed-version cluster (as might exist, for
+   example, during an upgrade), you might want to query the option schema from
+   a specific running daemon by running a command of the following form:
+
+.. prompt:: bash $
+
+   ceph daemon <name> config help [option]
 
 Runtime Changes
 ===============
 
 In most cases, Ceph permits changes to the configuration of a daemon at
-runtime. This can be used for increasing or decreasing the amount of logging
+run time. This can be used for increasing or decreasing the amount of logging
```

Comment:
IMHO conventionally this is one word.

Reply:
I think probably I'm going to accept that and bow to convention, even though doing so deprives us of an unambiguous distinction between the adjectival "runtime" and the obviously substantive "run time".

Reply:
I recall past arguments over "filesystem" where I lost :(

Reply:
I think this is a battle that goes on forever. I expect that sometime in 2024 or 2025 we will begin the process of looking at each of these cases individually.